packages/outstatic/src/pages/settings.tsx

```diff
@@ -1,16 +1,48 @@
-import { useContext } from 'react'
+import { useContext, useState } from 'react'
+import { clsx } from 'clsx'
```

Comment:
Could also be `classnames`, but `clsx` seems to perform better for simpler tailwind declaration merges